javascript/react-js/react-introduction.md

```diff
@@ -214,12 +214,17 @@ If you want to get a better understanding of how create-react-app works and whic
 
 Often React testing is done with `react-testing-library`. Taking your learnings from the Battleship project, you should apply these concepts going forward in all your future projects, even with React.
 
+### React Developer Tools
+
+React Developer Tools is an extension to the browser developer tools which allows you to inspect React components that build the page. It helps you to check and edit the React component tree as well as props, state, and hooks for each component. [This video](https://www.youtube.com/watch?v=rb1GWqCJid4) will show you how to install React Developer Tools and some basic debugging you can do with it.
+
 ### Assignment
 
 <div class="lesson-content__panel" markdown="1">
 1. Read the first couple of pages of the react.js documentation. Their documentation is a great resource for coming back at a later point if you have to get more familiar with certain concepts or have to fresh up something. So let's get started. Read [this](https://reactjs.org/docs/hello-world.html), [this](https://reactjs.org/docs/introducing-jsx.html), [this](https://reactjs.org/docs/rendering-elements.html) and [this](https://reactjs.org/docs/components-and-props.html) article. In the last one they already introduce the concept of props. Once again, don't worry too much about those yet.
 2. Watch [this video](https://www.youtube.com/watch?v=JPT3bFIwJYA&list=PL55RiY5tL51oyA8euSROLjMFZbXaV7skS) to get another quick explanation of React, and then watch these: ([one](https://www.youtube.com/watch?v=G40iHC-h0c0&list=PL55RiY5tL51oyA8euSROLjMFZbXaV7skS&index=4), [two](https://www.youtube.com/watch?v=9wK4gHoOh1g&list=PL55RiY5tL51oyA8euSROLjMFZbXaV7skS&index=5)) videos from the same series, which focus on components. Feel free to code along with the whole course if you enjoy it.
 3. Read [this](https://testing-library.com/docs/react-testing-library/intro) documentation on React testing.
+4. Do [these exercises](https://react-devtools-tutorial.vercel.app/element-selector-tool) to get familiar with what you can do with React Dev Tools.
```

Comment:
I'm not sure about adding these exercises. I would prefer to just see a link to the react dev tools docs if anything, but I think that the short section and video are probably enough IMO. I'd love other maintainers' opinions on this though.

Reply:
I do agree that those exercises might be a little over-whelming but I just thought it would be helpful to keep as reference. I'll remove the exercises for now. Would [this document](https://reactjs.org/blog/2014/01/02/react-chrome-developer-tools.html) be good to add to the assignment? if not, I can remove the assignment all together for now.

Reply:
Yeah that's just the page I had in mind when I mentioned the react docs.

What would you think about not having it be an assignment, but linking to that page in your section on the dev tools rather than the video you have currently linked? It seems to cover all the same core content in a shorter/non-video-only format, and it's first party to boot.

Reply:
Okay, I wont be adding anything to the assignment then. However, don't you think the react docs video is a little bit out-dated? it seems to mention react id on the dom node which is nowhere to be found on the current version of react dev-tools.

Also I found [non-video-resource](https://www.pluralsight.com/guides/debugging-components-with-react-developer-tools) that seems good. I might just add this to the section. Although, I also think adding this article along with the video I first mentioned would "visually" help.

Reply:
Ah, you're right, the video on the docs blogpost is pretty outdated so it probably isn't the best resource after all.